src/sentry/tasks/commits.py

```diff
@@ -245,14 +247,18 @@ def is_integration_provider(provider):
     return provider and provider.startswith("integrations:")
 
 
-def get_emails_for_user_or_org(user, orgId):
+def get_emails_for_user_or_org(user: RpcUser | None, orgId: int):
     emails = []
+    if not user:
+        return []
     if user.is_sentry_app:
         organization = Organization.objects.get(id=orgId)
-        members = organization.get_members_with_org_roles(roles=["owner"]).select_related("user")
+        members = organization.get_members_with_org_roles(roles=["owner"])
 
         for m in list(members):
-            emails.append(m.user.email)
+            user = user_service.get_user(user_id=m.user_id)
+            if user:
+                emails.append(user.email)
```

Comment:
This is going to be really expensive when we have an actual RPC here. Could we do a `user_service.get_many()` to get more efficient queries/requests?

Reply:
@markstory I used `user_service.get_many()` instead.